executor-service/src/main/java/executor/service/util/file/FileJsonParser.java

```diff
@@ -1,11 +1,12 @@
 package executor.service.util.file;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
-
 import java.io.File;
 import java.io.IOException;
 import java.util.List;
+import org.springframework.stereotype.Component;
 
+@Component
 public class FileJsonParser implements FileParser {
 
     private final ObjectMapper objectMapper;
```

Comment:
The `ObjectMapper` can be configured as a bean, this can be done in the configuration class.
```
 @Bean
    public ObjectMapper objectMapper() {
        return new ObjectMapper();
    }
```

Reply:
done, thanks